### DISCUSSION TITLE
How to track Learn content submission status?  Adding Asset leasing to the modules and update ADO Upload CSV file.

### DISCUSSION THREAD

**Discussion by AMarzoukG** [microsoft/dynamics365patternspractices#213] [Q&A] [answered]
I'm New to GitHub and i submitted  a content updated request via the form  Learn Content Submission.  the title was  Adding Asset leasing to the modules and update ADO Upload CSV file. i uploaded the attached document 
[Adding Asset leasing to the modules and update ADO Upload CSV file.docx](https://github.com/microsoft/dynamics365patternspractices/files/15440724/Adding.Asset.leasing.to.the.modules.and.update.ADO.Upload.CSV.file.docx)
Do i have to add the request as an issue. appreciate your response and glad that we have the chance to contribute and support the community

**Comment by rachel-profitt** [comment, marked as answer]
Hi AMarzoukG,
Thanks for bringing this to our attention, I was able to find the file and had a look. In the future, yes, please create an issue and choose the type Error. We don't currently have any automation around new files being attached so it is easy for this to fall through the cracks. I will work with @Harshad ***@***.***> who owns Acquire to Dispose to get this fixed in our next release.

Thanks again for your contributions!
Cheers,
Rachel


Rachel Profitt (She/her)

Principal Program Manager

Dynamics 365 R&D

________________________________
From: AMarzoukG ***@***.***>
Sent: Friday, May 24, 2024 5:22 PM
To: microsoft/dynamics365patternspractices ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [microsoft/dynamics365patternspractices] How to track Learn content submission status? Adding Asset leasing to the modules and update ADO Upload CSV file. (Discussion #213)


I'm New to GitHub and i submitted a content updated request via the form Learn Content Submission. the title was Adding Asset leasing to the modules and update ADO Upload CSV file. i uploaded the attached document
Adding Asset leasing to the modules and update ADO Upload CSV file.docx<https://github.com/microsoft/dynamics365patternspractices/files/15440724/Adding.Asset.leasing.to.the.modules.and.update.ADO.Upload.CSV.file.docx>
Do i have to add the request as an issue. appreciate your response and glad that we have the chance to contribute and support the community

**Reply by AMarzoukG** [reply]
Thanks so much @rachel-profitt   for the quick response. 
Is this considered a contribution that supports the new badges system ?
